internal/pkg/telemetry/fortest.go

```diff
@@ -32,8 +33,8 @@ const (
 
 type ForTest interface {
 	Telemetry
-	TraceID(n int) trace.TraceID
-	SpanID(n int) trace.SpanID
+	TraceID(n uint16) trace.TraceID
+	SpanID(n uint16) trace.SpanID
```

Comment:
I don't like 16 bit width variables. What happens if overflows? Returns error and Span, Trace will not be generated?

Reply:
I need more time for this

Reply:
https://github.com/keboola/keboola-as-code/pull/2065/commits/c7e95b3c795433ace4dbf0ff970ad3090fbad58e